components/controls/buttons.tsx

```diff
@@ -74,8 +76,7 @@ export class Button extends React.PureComponent<ButtonProps> {
       // eslint-disable-next-line react/button-has-type
       <button
         {...props}
-        className={classNames('button', className)}
-        disabled={this.props.disabled}
```

Comment:
As discussed directly, I think we may have an issue since the disabled state is style only.
Can you add https://www.w3.org/WAI/PF/aria/states_and_properties#aria-disabled to bring a meaningful state to the button ? Thanks !